### PR TITLE
chore: use externally published cloud-assembly-schema

### PR DESCRIPTION
**Pull request by TheRealAmazonKendra**
### Issue # (if applicable)

Closes #<issue number here>.

### Checklist
- [ ] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/integ-runner/package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
   },
   "dependencies": {
     "chokidar": "^3.6.0",
-    "@aws-cdk/cloud-assembly-schema": "0.0.0",
+    "@aws-cdk/cloud-assembly-schema": "^36.0.1",
     "@aws-cdk/cloudformation-diff": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
```
Comment (hanuytayal) [inline packages/@aws-cdk/integ-runner/package.json:76]: Amazon Q: Please use the correct version as shown below
```suggestion
    "@aws-cdk/cx-api": "^37.0.1",
```

Comment (TheRealAmazonKendra) [reply packages/@aws-cdk/integ-runner/package.json:76]: This version of cx-api doesn't exist.

**Comment by TheRealAmazonKendra** [comment]
Adding the do-not-merge label until this makes its way though the testing pipeline.

**Review by scanlonp — APPROVED** [review]

**Comment by TheRealAmazonKendra** [comment]
@Mergifyio update

**Pull request merged**